hadoop-hdds/docs/content/security/GDPR.md

```diff
@@ -2,7 +2,6 @@
 title: "GDPR in Ozone"
 date: "2019-September-17"
 weight: 3
-summary: GDPR in Ozone
```

Comment:
Why is this line removed?

Reply:
> Why is this line removed?

@kaijchen  `summary ` repeated.
The line deleted was the same as the `title`, so I kept the latter.
```js
summary: GDPR in Ozone
icon: user
menu:
   main:
      parent: Security
summary: Support to implement the "Right to be Forgotten" requirement of GDPR
```